WF camera/ Updates

AArlauskas opened the pull request:
Name changes
Variable naming fixes
Saved Image file naming fixed
Default resoliution set to maximum
Output camera view is fixed to window size

AArlauskas committed:
I renamed project I hope it works

AArlauskas committed:
test

Pilkas-git commented:
Requested changes approved.

Pilkas-git reviewed (approved):
Code changed following requests.

tmaciulis22 commented on Moodis/Moodis/cameraForm.cs:
```diff
@@ -0,0 +1,100 @@
+
+namespace moodie
+{
+    public partial class cameraForm : Form
```
Class names should start with capital letters

tmaciulis22 commented on Moodis/Moodis/cameraForm.cs:
```diff
@@ -0,0 +1,100 @@
+        private void CameraFormLoad(object sender, EventArgs e)
+        {
+            webcam = new FilterInfoCollection(FilterCategory.VideoInputDevice);
+            foreach(FilterInfo VideoCaptureDevice in webcam)
```
change FilterInfo to var

Pilkas-git replied:
I don't believe he can change that based on the documentation www.aforgenet.com/framework/docs/html/8d13e608-64f7-8c03-84b1-1bce74dabecd.htm

tmaciulis22 commented on Moodis/Moodis/cameraForm.cs:
```diff
@@ -0,0 +1,100 @@
+
+            cam = new VideoCaptureDevice(webcam[cmbOutputDevices.SelectedIndex].MonikerString);
+
+            foreach (VideoCapabilities option in cam.VideoCapabilities)
```
same here, change to var

tmaciulis22 commented on Moodis/Moodis/cameraForm.cs:
```diff
@@ -0,0 +1,100 @@
+
+                }
+            }
+
```
Empty line

tmaciulis22 commented on Moodis/Moodis/cameraForm.cs:
```diff
@@ -0,0 +1,100 @@
+                cam.VideoResolution = cam.VideoCapabilities[cmbCamResoliution.SelectedIndex];
+                cam.Start();
+            }
+            
```
Empty line

tmaciulis22 reviewed (changes requested):
Minor changes, but all in all good job 👍

Pilkas-git commented on Moodis/Moodis/Moodis.csproj:
```diff
@@ -0,0 +1,97 @@
+    <ProjectGuid>{B5CEA781-5F92-4EDF-BC87-CF352F9AD6BF}</ProjectGuid>
+    <OutputType>WinExe</OutputType>
+    <RootNamespace>Moodie</RootNamespace>
+    <AssemblyName>Moodie</AssemblyName>
```
Change both of "Moodie" to Moodis.

Pilkas-git commented on Moodis/Moodis/Program.cs:
```diff
@@ -0,0 +1,22 @@
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace moodie
```
moodis not moodie. This is also going to edit a lot of references so be careful when renaming.

Pilkas-git commented on Moodis/Moodis/Properties/AssemblyInfo.cs:
```diff
@@ -0,0 +1,36 @@
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("moodie")]
```
Change title to moodis.

Pilkas-git commented on Moodis/Moodis/Properties/AssemblyInfo.cs:
```diff
@@ -0,0 +1,36 @@
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("moodie")]
```
Change to moodis.

Pilkas-git commented on Moodis/Moodis/cameraForm.cs:
```diff
@@ -0,0 +1,100 @@
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace moodie
```
moodis not moodie. This is also going to edit a lot of references so be careful when renaming.

AArlauskas committed:
FixedCodeStyle

Fixed empty lines, moodis naming and desired variables to var(Only those that are supported).

Pilkas-git commented:
Requested changed done in latest commit.

AArlauskas committed:
Reupload of fixes

Missed one empty line, now fixed :)

Pilkas-git reviewed (approved):
Completed all of the requested changes. Approved
